test/OBSSStorage.KetlKred.test.ts

```diff
@@ -0,0 +1,114 @@
+import { Feeds, KetlCred, OBSSStorage, Profiles } from '../typechain'
+import { MOCK_CID, zeroAddress } from './utils'
+import { ethers, upgrades } from 'hardhat'
+import { expect } from 'chai'
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers'
+import { smock } from '@defi-wonderland/smock'
+import { version } from '../package.json'
+
+describe('OBSSStorage: KetlKred', () => {
+  async function setupOBSS() {
+    const [deployer, user1, user2, user3] = await ethers.getSigners()
+
+    const ketlAttestationContractMock = await smock.fake([
+      {
+        inputs: [
+          { internalType: 'address', name: 'account', type: 'address' },
+          { internalType: 'uint256', name: 'id', type: 'uint256' },
+        ],
+        name: 'balanceOf',
+        outputs: [{ internalType: 'uint256', name: '', type: 'uint256' }],
+        stateMutability: 'view',
+        type: 'function',
+      },
+      {
+        inputs: [],
+        name: 'currentTokenId',
+        outputs: [{ internalType: 'uint32', name: '', type: 'uint32' }],
+        stateMutability: 'view',
+        type: 'function',
+      },
+    ])
+    ketlAttestationContractMock.balanceOf.returns(1)
+    ketlAttestationContractMock.currentTokenId.returns(1)
+
+    const profilesContractFactory = await ethers.getContractFactory('Profiles')
+    const profilesContract = (await upgrades.deployProxy(
+      profilesContractFactory,
+      [ketlAttestationContractMock.address, 0, deployer.address]
+    )) as Profiles
+    await profilesContract.deployed()
+
+    const ketlCredContractFactory = await ethers.getContractFactory('KetlCred')
+    const ketlCredContract = (await upgrades.deployProxy(
+      ketlCredContractFactory,
+      ['Ketl', 'KETL', 0, deployer.address],
```

Comment:
As I remember, we used `'kekl', 'KEK'` for testing, maybe we should use it here as well